src/config/stfc/setStfcColourTheme.ts

```diff
@@ -0,0 +1,25 @@
+import { container } from 'tsyringe';
+
+import { AdminDataSource } from '../../datasources/AdminDataSource';
+import { SettingsId } from '../../models/Settings';
+import { Tokens } from '../Tokens';
+
+export default function setStfcColourTheme() {
+  const dataSource = container.resolve<AdminDataSource>(Tokens.AdminDataSource);
+  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_DARK, '#303f9f');
+  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_MAIN, '#3f51b5');
+  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_LIGHT, '#7986cb');
+  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_CONTRAST, '#ffffff');
+  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_DARK, '#c51162');
+  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_MAIN, '#f50057');
+  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_LIGHT, '#ff4081');
+  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_CONTRAST, '#ffffff');
+  dataSource.updateSettings(SettingsId.PALETTE_ERROR_MAIN, '#f44336');
+  dataSource.updateSettings(SettingsId.PALETTE_SUCCESS_MAIN, '#4caf50');
+  dataSource.updateSettings(SettingsId.PALETTE_WARNING_MAIN, '#ff9800');
+  dataSource.updateSettings(SettingsId.PALETTE_INFO_MAIN, '#2196f3');
```

Comment:
Looks great, thanks for doing this! I've put the STFC colours below, as what's there currently is the default Material UI colours

```suggestion
  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_DARK, '#2e2d62');
  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_MAIN, '#003088');
  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_LIGHT, '#1e5df8');
  dataSource.updateSettings(SettingsId.PALETTE_PRIMARY_CONTRAST, '#ffffff');
  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_DARK, '#d77900');
  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_MAIN, '#ff9d1b');
  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_LIGHT, '#fbbe10');
  dataSource.updateSettings(SettingsId.PALETTE_SECONDARY_CONTRAST, '#000000');
  dataSource.updateSettings(SettingsId.PALETTE_ERROR_MAIN, '#a91b2e');
  dataSource.updateSettings(SettingsId.PALETTE_SUCCESS_MAIN, '#3e863e');
  dataSource.updateSettings(SettingsId.PALETTE_WARNING_MAIN, '#fbbe10');
  dataSource.updateSettings(SettingsId.PALETTE_INFO_MAIN, '#1e5df8');
```